File: SIPClient/app/src/main/java/scn/com/sipclient/realm/HistoryModel.java
package scn.com.sipclient.realm;

import io.realm.RealmObject;

/**
 * Created by pakjs on 9/4/2016.
 */
public class HistoryModel extends RealmObject {
    private int HistoryId;
    private String CallerEmail;
    private int CallStatus;
    private String CallDate;
    private String CallSummary;

    public String getCallDate() {
        return CallDate;
    }

    public void setCallDate(String callDate) {
        CallDate = callDate;
    }

    public String getCallerEmail() {
        return CallerEmail;
    }

    public void setCallerEmail(String callerEmail) {
        CallerEmail = callerEmail;
    }

    public int getCallStatus() {
        return CallStatus;
    }

    public void setCallStatus(int callStatus) {
        CallStatus = callStatus;
    }

    public String getCallSummary() {
        return CallSummary;
    }

    public void setCallSummary(String callSummary) {
        CallSummary = callSummary;
    }

    public int getHistoryId() {
        return HistoryId;
    }

    public void setHistoryId(int historyId) {
        HistoryId = historyId;
    }
}

File: SIPAPI/WebApplication4/Models/HistoryModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace WebApplication4.Models
{
    public class HistoryModel
    {
        [Key]
        public int HistoryId { get; set; }
        public string CallerEmail { get; set; }
        public int CallStatus { get; set; }      // 0: Call in,  1: Call out
        public string CallSummary { get; set; }
        public string CallDate { get; set; }
    }
}